### PR TITLE
DMN Runner Fixes - Pt2

### PR DESCRIPTION
**Pull request by ljmotta**
## On this PR
Improved general code base:
 - Persistence of the column width;
 - Using useReducer to pass down the dmn persistence JSON. All checks and FS update is done inside the reducer function;
 - Moved dmnRunner files to online-editor top level;
 - Rename DmnRunnerInputs to DmnRunnerPersistence;
 - Add a debouncer to perform less FS updates;

### CODE REVIEW

File: packages/unitables/src/bee/UnitablesBeeTable.tsx [hunk 1]
```diff
@@ -54,6 +55,8 @@ export interface UnitablesBeeTable {
   onRowDuplicated: (args: { rowIndex: number }) => void;
   onRowReset: (args: { rowIndex: number }) => void;
   onRowDeleted: (args: { rowIndex: number }) => void;
+  rowsWidth: object;
```
Comment (tiagobento) [inline packages/unitables/src/bee/UnitablesBeeTable.tsx:58]: Not sure this is ideal...

File: packages/unitables/src/bee/UnitablesBeeTable.tsx [hunk 2]
```diff
@@ -32,12 +32,13 @@ import * as React from "react";
 import { useCallback, useMemo } from "react";
 import * as ReactTable from "react-table";
 import { UnitablesColumnType } from "../UnitablesTypes";
-
 import "@kie-tools/boxed-expression-component/dist/@types/react-table";
 import { ResizerStopBehavior } from "@kie-tools/boxed-expression-component/dist/resizing/ResizingWidthsContext";
 import { AutoField } from "@kie-tools/uniforms-patternfly/dist/esm";
 import { useField } from "uniforms";
 import { AUTO_ROW_ID } from "../uniforms/UnitablesJsonSchemaBridge";
+import get from "lodash/get";
```
Comment (tiagobento) [inline packages/unitables/src/bee/UnitablesBeeTable.tsx:40]: Shouldn't we be doing `_.get`? I mean, it's weird to have a `get` floating around :D

**Review by tiagobento — APPROVED** [review]
Approved and merged, but please take a look at the comments when you have a chance.

File: packages/unitables/src/bee/UnitablesBeeTable.tsx (hunk 1 above)
Comment (ljmotta) [reply packages/unitables/src/bee/UnitablesBeeTable.tsx:58]: It's not. I forgot to change the type, the correct one for now is `ConfigInputRow`. It's not the ideal name, I'll change this too

File: packages/unitables/src/bee/UnitablesBeeTable.tsx (hunk 2 above)
Comment (ljmotta) [reply packages/unitables/src/bee/UnitablesBeeTable.tsx:40]: As I'm using the default export, I can change the name of it for what we want. Maybe `getObjectValueByPath` is a better name.

**Pull request merged**